tool/test.sh

```diff
@@ -4,7 +4,7 @@ set -x
 
 if [[ "${FLUTTER}" = true ]]
 then
-  $FLUTTER_CMD test --coverage --coverage-path coverage/lcov.info test/all.dart
+  $FLUTTER_CMD test --coverage --coverage-path coverage/lcov.info
```

Comment:
Why was `all.dart` removed?

Reply:
`test_coverage` is not used on flutter packages. So all that this `all.dart` file did was simulating the default behavior of the command line.

On the other hand this `all.dart` causes issue if we add new test files and forget to add them in this file.

Reply:
`test_coverage` generates it's own combined file. I've used the `all.dart` to run all unit tests with a single Dart VM launch which usually makes them run faster than every file opening their own separate VM.